api/handlers_test.go

```diff
@@ -0,0 +1,275 @@
+package api_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/nhatvu148/business-day-go/api"
+)
+
+func cleanupDB(t testing.TB, app *api.Application) {
+	req := httptest.NewRequest(http.MethodDelete, "/api/v1/custom-holiday", nil)
```

Comment:
Use [t.Healper](https://pkg.go.dev/testing#T.Helper) to get cleaner stack traces from these functions.